lib/galaxy/datatypes/converters/xtc_or_trr_to_dcd.xml

```diff
@@ -0,0 +1,35 @@
+<tool id="CONVERTER_Xtc_or_Trr_to_Dcd_0" name="XTC or TRR to DCD" version="1.0.0" hidden="true">
+    <requirements>
+        <requirement type="package" version="1.9.4">mdtraj</requirement>
+    </requirements>
+    <command detect_errors="exit_code"><![CDATA[
+        ln -s '$input1' ./input.${input1.ext} &&
+        mdconvert ./input.${input1.ext} -o ./output.dcd 
+    ]]></command>
+    <inputs>
+        <param format="xtc,trr" name="input1" type="data" label="Choose input file"/>
+    </inputs>
+    <outputs>
+        <data format="dcd" name="output1" from_work_dir="output.dcd"/>
+    </outputs>
+    <tests>
+        <test>
+            <param name="input1" format="xtc" value="traj.xtc"/>
+            <output name="output1" format="dcd">
```

Comment:
The dcd files have a time stamp in the binary which is a bit annoying. I don't know of a better way to compare then the file size.